cogs/event_listener.py

```diff
@@ -23,7 +23,7 @@ async def on_message(self, message: discord.Message):
         logging.debug('Message from {0.author}: {0.content}'.format(message))
 
         # Ignore message if it was from a bot
-        if message.author == bot.user:
+        if message.author.bot:
```

Comment:
Why is this being changed? Does this change in master already solve the issue that this change is meant to do? https://github.com/mbox-contributors/mbox/blob/9110e6c8cc36f200d187c755df70e20e7fb264c2/cogs/event_listener.py#L37